renderer/views/app.js

```diff
@@ -11,8 +11,14 @@ var TorrentList = require('./torrent-list')
 var isOSX = process.platform === 'darwin'
 
 function App (state, dispatch) {
+  var cls = [
+    process.platform,
+    state.isFullScreen ? 'fullscreen' : 'not-fullscreen',
+    state.url === '/player' ? 'view-player' : ''
+  ]
+
   return hx`
-    <div class='app'>
+    <div class='app ${cls.join(' ')}'>
```

Comment:
I was thinking in the long run leaning on CSS classes more to show/hide will be a lot easier. :+1: